lib/metrics.go

```diff
@@ -140,8 +143,9 @@ func (m *Metrics) init() {
 	if m.latencies == nil {
 		m.latencies = quantile.New(
 			quantile.Known(0.50, 0.01),
-			quantile.Known(0.95, 0.001),
-			quantile.Known(0.99, 0.0005),
+			quantile.Known(0.90, 0.001),
```

Comment:
What was the thought process behind these tolerance values?

Reply:
See my original PR comment for background to this, I did ask for review of the implementation. But to answer your question, naive gut instinct, based on the existing implementation. I don't know if it's correct, but seems close enough? This kind of implementation to a math novice is less straightforward than typical statistical percentile libraries. What should be the "proper" values? And while we're at this, how to calculate the tolerance values for any given percentile in case other users might want something else like 75-80th percentiles (relative to the current ones).